core/tests/test_utils.py

```diff
@@ -1731,6 +1732,31 @@ def get_static_asset_url(self, asset_suffix):
         """
         return '/assets%s%s' % (utils.get_asset_dir_prefix(), asset_suffix)
 
+    @contextlib.contextmanager
+    def mock_datetime_utcnow(self, mocked_datetime):
+        original_datetime_type = datetime.datetime
+
+        class PatchedDatetimeType(type):
+            """Validates the datetime instances."""
+            def __instancecheck__(cls, other):
+                """Validates whether the given instance is a datatime
```

Comment:
`...is datetime...`

Reply:
@vpishuk -- The spelling mistake is still there.

Reply:
ah, I see. Fixed